src/mbgl/style/expression/expression.cpp

```diff
@@ -18,12 +20,8 @@ class GeoJSONFeature : public GeometryTileFeature {
     const PropertyMap& getProperties() const override { return feature.properties; }
     FeatureIdentifier getID() const override { return feature.id; }
     GeometryCollection getGeometries() const override { return {}; }
-    optional<mbgl::Value> getValue(const std::string& key) const override {
-        auto it = feature.properties.find(key);
-        if (it != feature.properties.end()) {
-            return optional<mbgl::Value>(it->second);
-        }
-        return optional<mbgl::Value>();
+    mbgl::Value getValue(const std::string& key) const override {
+        return feature.properties.count(key) ? feature.properties.at(key) : NullValue();
```

Comment:
Let's not use this pattern; it does the same lookup twice, while the previous code only did it once.

Reply:
My bad - forgot to update these.